### DISCUSSION TITLE
File Permissions

### DISCUSSION THREAD

**Discussion by Bob-AOL** [gtxaspec/wz_mini_hacks#430] [Q&A] [answered]
Finally got SSH going to edit wz_mini.conf live in the slot in hopes of avoiding the possibility that editing otherwise has been causing  recent difficulty successfully enabling the USB/Ethernet feature with known good UCTRONICS adapters. Should take 5 minutes to set one up and has in the past. I'm now going on 3 days with my latest two v3s using the same method as before with zero success this time around. Two known good adapters, cables, all that stuff covered.

So I get my public keys working and talking to the cams from both Windows and Ubuntu. Got refreshed a bit on Nano, enabled the USB/ethernet capability along with setting up the cam and RTSP params and all was good until I tried to save the file only to discover a read-only filesystem that root doesn't even seem to have permission to change. ls -l seems to show root owning everything with all permissions for all folders and files. 

I can pull the card and edit and save all day long in Notepad and also via the web interface, but no dice live in the cam with SSH as has been so highly recommended as the right way to avoid problems from day one. So what am I missing?

As usual folks, TIA for any enlightenment.

**Comment by Bob-AOL** [comment, marked as answer]
Fixed permissions. Wz_mini still broken.

**Comment by mesvam** [comment]
I used `mount -o remount,rw /dev/mmcblk0p1 /opt` to make it writable

**Comment by dasb00ter** [comment]
hmm this happened to me saved my a** @mesvam
